nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -431,8 +432,32 @@ contract StakingEscrow is Issuer {
     }
 
     //------------------------Main methods------------------------
+
+    /**
+    * @notice Update bounds of allowed locked tokens if values are less than previously defined
+    * @dev Only from worklock contract
+    * @param _min New value for `minAllowableLockedTokens`
+    * @param _max New value for `maxAllowableLockedTokens`
+    */
+    function updateAllowableLockedTokens(uint256 _min, uint256 _max) external {
+        require(msg.sender == address(workLock));
+        bool updated = false;
+        if (minAllowableLockedTokens > _min) {
+            minAllowableLockedTokens = _min;
+            updated = true;
+        }
+        if (maxAllowableLockedTokens > _max) {
```

Comment:
So we can only change the max if it less than the previous max - what's the reasoning here?

Reply:
Yep, we can only reduce max and min to keep ratio between them. Because we should decrease min stake size if bidder with lowest bid will get less tokens than default min. And also we should keep ratio between max and min stake sizes otherwise could be issues with granting. In the result - we can only reduce

Reply:
👍